packages/core/useFileDownload/index.ts

```diff
@@ -0,0 +1,38 @@
+export interface DownloadOptions {
+  href: string | Blob
+  fileName: string
```

Comment:
how about `content-disposition`header? 

there are situations where you don't know the filename, for example using an api to download some doc for some id...

Firefox will handle attachments in a diferent way chromium based browsers, I use this code when the response is an attachement:
```ts
      const contentDisposition = headers.get("content-disposition");
      // const inline = contentDisposition.indexOf("inline;") > -1;
      const matches = /filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/.exec(
        contentDisposition
      );
      const fileName = matches && matches[1].replace(/['"]/g, "");
      // FIX: force download the document: the browser can open it or not (ff does but chrome doesn't)
      const contentType = "application/octet-stream"; // headers.get("content-type");
      link.setAttribute("download", fileName);
      await nextTick();
      link.click();     
```